.github/workflows/destroy.yml

```diff
@@ -1,6 +1,12 @@
 name: Destroy
 
-on: delete
+on:
+  delete:
+  workflow_dispatch:
```

Comment:
This will enable the manual execution of destroy against any given stage without needing to locate the stage to destroy it.  It maintains the checks that prevent destruction of any of the integration states.  One small issue though, is that again, this has to execute from the default branch, so this has to be merged in to test.